rviz_default_plugins/src/rviz_default_plugins/displays/wrench/wrench_display.cpp

```diff
@@ -51,6 +51,10 @@ namespace displays
 
 WrenchDisplay::WrenchDisplay()
 {
+  accept_NaN_values_ = new rviz_common::properties::BoolProperty(
+    "Accept NaN Values", true,
```

Comment:
Please set it default to `false` so we keep current behavior.

```suggestion
    "Accept NaN Values", false,
```